M2/Macaulay2/m2/modules.m2

```diff
@@ -249,6 +242,15 @@ toString Module := M -> toString expression M
 net Module := M -> net expression M
 texMath Module := M -> texMath expression M
 
+-- returns null if can't shorten (free module or assigned to variable)
+moduleAbbrv = M -> (
+    if isFreeModule M and hasAttribute(ring M, ReverseDictionary)
+    then expression M
+    else if hasAttribute(M, ReverseDictionary)
+    then getAttribute(M, ReverseDictionary))
+
+short Module := M -> moduleAbbrv M ?? expression M
```

Comment:
I don't understand why you removed the second input of `moduleAbbrv`, when this could have been implemented as:
```suggestion
short Module := M -> moduleAbbrv(M, expression M)
```

Reply:
I've always thought that `moduleAbbrv`'s two inputs were kind of clunky.  For example, in `Matrix.AfterPrint`, we don't need the second input at all -- if we can't abbreviate the module, then we just don't print anything.  I figured this was an opportunity to update it.